index.html

```diff
@@ -22,16 +22,44 @@
             </nav>
         </header>
         <section class="hero">
-            <h1>Hector<br>Pereira</h1>
-            <p>Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
-            <button>
-                <span>Start collaboration</span>
-            </button>
+            <div class="hero-content">
+                <h1>Hector<br>Pereira</h1>
+                <p>Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+                <button>
+                    <span>Start collaboration</span>
+                </button>
+            </div>
```

Comment:
Reduce the image size in this section so it only fits the main page 
### current image size
![wrong1](https://user-images.githubusercontent.com/58376020/121605759-38d97500-ca4d-11eb-87d3-77113c025c1f.png)

## Original design
![right2](https://user-images.githubusercontent.com/58376020/121605822-57d80700-ca4d-11eb-8655-b7b571669294.png)

Reply:
fixed 👍

Reply:
Please notice that using the vw unit in the width property is causing an overflow on the website. Kindly keep in mind that vw stands for "viewport width" and this means that this element is going to take all the available space on the screen including the document margin. Please use the % unit instead, in order to fix the design issue.

using vw unit is making the background images bigger causing it to over flow.
![Screenshot (50)](https://user-images.githubusercontent.com/52497246/121759719-0f891980-cb1f-11eb-8a85-045a0b17b24b.png)
![Screenshot (51)](https://user-images.githubusercontent.com/52497246/121759723-16b02780-cb1f-11eb-9453-922825938e03.png)